testing/tests/DevExpress.core/utils.resize_callbacks.tests.js

```diff
@@ -86,6 +87,26 @@ QUnit.test('Callback is not called if size is not changed', function(assert) {
     assert.ok(!called, 'callback is not called');
 });
 
+QUnit.test('Callback is called if window innerHeight is changed (T834502)', function(assert) {
+    let callCount = 0;
+    this.callbacks.add(function() {
+        callCount++;
+    });
+    const originalGetWindow = windowUtils.getWindow;
+
+    try {
+        windowUtils.getWindow = function() {
+            return { innerHeight: 100, innerWidth: 100 };
+        };
+
+        this.triggerResize(false);
+
+        assert.strictEqual(callCount, 1, 'callback is called');
+    } finally {
+        windowUtils.getWindow = originalGetWindow;
```

Comment:
Remove the test callback added [here](https://github.com/DevExpress/DevExtreme/pull/11070/files#diff-c2fcb0648b6ef84d628bc321b8d1eae7R92)?